packages/hash/api/src/seed-data/seed-pages.ts

```diff
@@ -0,0 +1,73 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { PageModel } from "../model";
+
+const createNestedPages = async (
+  owningActorId: string,
+  [top, nested]: [string, string],
+  prevIndex: string | undefined,
+  sharedParams: {
+    graphApi: GraphApi;
+    logger: Logger;
+  },
+): Promise<PageModel> => {
+  const { graphApi } = sharedParams;
+
+  const topPageModel = await PageModel.createPage(graphApi, {
+    actorId: owningActorId,
+    ownedById: owningActorId,
+    title: top,
+    prevIndex,
+  });
+
+  const nestedPageModel = await PageModel.createPage(graphApi, {
+    actorId: owningActorId,
+    ownedById: owningActorId,
+    title: nested,
+  });
```

Comment:
Nit: we can create these concurrently

Reply:
Thanks, done in 6771bc53881ab35fb715783fcccbbf2b7677802e :)